.github/workflows/add-to-dev-prod-project.yml

```diff
@@ -9,6 +9,9 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.BOT_GRADLE_GH_ORG_BOARD_ACCESS_TOKEN }}
 
+permissions:
+  contents: read
```

Comment:
Hi @wolfs, this workflow uses the `secrets.BOT_GRADLE_GH_ORG_BOARD_ACCESS_TOKEN` secret, which probably has access to the org-wide project. 

The permissions being set in the PR are for the `GITHUB_TOKEN`. That token has excessive permissions (see https://github.com/gradle/gradle/runs/7182789855?check_suite_focus=true#step:1:19), but is not being used in the workflow. 

Moreover, `GITHUB_TOKEN` does not have permissions to the org-wide project. It can only be used for repository projects. So `GITHUB_TOKEN` cannot be used for your scenario in this workflow. So, reducing the permissions for `GITHUB_TOKEN` should be fine here.